src/slim-select/index.test.ts

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
I love that you were able to add tests. I hate front end testing with a passion.